tests/ui/deriving/deriving-all-codegen.stdout

```diff
@@ -126,7 +126,7 @@ impl ::core::marker::StructuralPartialEq for Point { }
 impl ::core::cmp::PartialEq for Point {
     #[inline]
     fn eq(&self, other: &Point) -> bool {
-        self.x == other.x && self.y == other.y
+        self.y == other.y && self.x == other.x
```

Comment:
It's unfortunate that the order is "scalar fields in reverse order, then non-scalar fields in reverse order". Is it possible to do things in non-reverse order?